docs/rfcs/1014-account-2-electric-boogaloo.md

```diff
@@ -0,0 +1,705 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# Account 2: Electric Boogaloo
+
+## 1 - Goals
+
+1. Simplify user interaction by making Parsec account a one-size-fits-all solution
+2. Prevent Device & user keys form being stored on the user machine disk.
+3. Make possible the use of a secure way (HSM) to store the data protected by lower entropy secrets (e.g. password).
+
+Key points:
+
+- Parsec client always ask for Parsec account login
+- Offline login is possible
```

Comment:
It would be nice to have a explicit definition of "offline login" somewhere.

> Offline in regards to parsec account but logged into an organization. To not be confused with the current offline mode when there is no connection to the metadata server

By the way how does it interact with offline offline mode ? There would be 4 possible state ?
- connected to parsec account and parsec metadata
- not connected to parsec account but connected to parsec metadata
- connected to neither
- connected to parsec account but not parsec metadata should occur only if there is an issue with the metadata sever ? It shouldn't be that different from the offline offline mode ?